src/routes/console/project-[project]/auth/user-[user]/updateLabels.svelte

```diff
@@ -0,0 +1,97 @@
+<script lang="ts">
+    import { invalidate } from '$app/navigation';
+    import { Submit, trackEvent, trackError } from '$lib/actions/analytics';
+    import { CardGrid, Heading } from '$lib/components';
+    import { Dependencies } from '$lib/constants';
+    import { Button, Form, Helper, InputTags } from '$lib/elements/forms';
+    import { addNotification } from '$lib/stores/notifications';
+    import { sdk } from '$lib/stores/sdk';
+    import { user } from './store';
+    import { Pill } from '$lib/elements';
+
+    const alphaNumericRegExp = /^[a-zA-Z0-9]+$/;
+    let suggestedLabels = ['admin', 'premium', 'mvp'];
+    let labels = [...$user.labels];
```

Comment:
I wonder if we should set this also on mount :thinking: If I remember correctly we did it on other pages because when you reloaded the page sometimes the data wasn't set correctly, not sure if it's needed here though